developer-docs-site/static/examples/python/first_nft.py

```diff
@@ -121,23 +122,33 @@ def cancel_token_offer(
         self.submit_transaction_helper(account, payload)
             
 #:!:>section_3
+    def table_item(self, handle: str, key_type: str, value_type: str, key: Any) -> Any:
```

Comment:
Don't we have any tests for these tutorials which is why we were able to land a change to break them? If so, is that something we should follow up on?

Reply:
There may have been tests, but we've been flaking out in CI/CD on docker containers.

Reply:
I deliberately send these changes separately. 
In theory these work with the live network which might not be compatible with the trunk code, so them being separate or "decoupled" makes sense to me. 
Would be nice to have a easier way to know they are broken / started to be incompatible though.